drivers/node/node.go

```diff
@@ -20,23 +20,32 @@ const (
 
 // Node encapsulates a node in the cluster
 type Node struct {
-	Name      string
-	Addresses []string
-	Type      Type
+	Name       string
+	Addresses  []string
+	UsableAddr string
+	Type       Type
+}
+
+// ConnectionOpts provide basic options for all operations and can be embedded by other options
+type ConnectionOpts struct {
+	Timeout         time.Duration
+	TimeBeforeRetry time.Duration
 }
 
 // RebootNodeOpts provide additional options for reboot operation
 type RebootNodeOpts struct {
-	Force bool
+	Force          bool
+	ConnectionOpts ConnectionOpts
```

Comment:
Use an anonymous filed here. So something like
```
type RebootNodeOpts struct {
 	Force bool
        ConnectionOpts
}
```
So users can use it as ops.Timeout instead of ops.ConnectionOpts.Timeout.